### PR TITLE
Add info about how to install and test library on local machine

### PR DESCRIPTION
**Pull request by sahildua2305**
Added `composer install` and `composer test` commands to facilitate the future contributors

/cc @frederikbosch @sagikazarmark

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,4 +81,4 @@ contributions, in code and via his blog, this library would not be where it stan
 [2]: https://github.com/pink-tie/MoneyBundle/
 [3]: http://symfony.com/
 [4]: https://github.com/TheBigBrainsCompany/TbbcMoneyBundle
-[5]: https://packagist.org/packages/mathiasverraes/money
```
Comment (sagikazarmark) [inline README.md]: What happened here?

Comment (sahildua2305) [inline README.md]: I'm not sure. GitHub bug may be. I tried this twice. Didn't change anything on this line. It still shows this line in diff.

It's same anyway.

File: README.md [hunk 2]
```diff
@@ -60,6 +60,12 @@ and prefixes in class names, so not `CurrenciesInterface` but `Currencies`. Othe
 using the same code style as we are using. Please provide tests when creating a PR and clear descriptions of bugs when
 filing issues.
 
+You can run the following commands to get started:
+```
```
Comment (sagikazarmark) [inline README.md]: bash

File: README.md [hunk 3]
```diff
@@ -60,6 +60,12 @@ and prefixes in class names, so not `CurrenciesInterface` but `Currencies`. Othe
...
+composer install
```
Comment (sagikazarmark) [inline README.md]: $

File: README.md [hunk 4]
```diff
@@ -60,6 +60,12 @@ and prefixes in class names, so not `CurrenciesInterface` but `Currencies`. Othe
 using the same code style as we are using. Please provide tests when creating a PR and clear descriptions of bugs when
 filing issues.
 
+You can run the following commands to get started:
```
Comment (sagikazarmark) [inline README.md]: Can you add a Testing heading? Like here: https://github.com/php-http/httplug

File: README.md (hunk 3 above)
Comment (sagikazarmark) [inline README.md]: I don't think this is necessary, it's kind of obvious.

Comment (sahildua2305) [inline README.md]: The terminal sign (`$`)? Yes, that's why I didn't add it at first place. I'll add it though. Makes more sense.

File: README.md (hunk 4 above)
Comment (sahildua2305) [inline README.md]: Sure!

File: README.md (hunk 2 above)
Comment (sahildua2305) [inline README.md]: Oh yeah! Thanks :)

File: README.md [hunk 5]
```diff
@@ -52,6 +52,13 @@ Integration
 
 See [`MoneyBundle`][2] or [`TbbcMoneyBundle`][4] for [Symfony integration][3].
 
+Testing
+-------
+```bash
+$ composer install
```
Comment (sagikazarmark) [inline README.md]: Sorry, I meant composer install here. For me, it's quite obvious

Comment (sahildua2305) [inline README.md]: Oh okay. My bad. Makes sense. I'll remove it.

**Comment by sagikazarmark** [comment]
Thanks @sahildua2305 , will merge it as soon as CI is ready.

**Comment by sahildua2305** [comment]
@sagikazarmark Looks like it's ready :smile:

**Pull request merged**